Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -5023,3 +5024,35 @@ void CClientVehicle::ResetWheelScale()
 
     m_bWheelScaleChanged = false;
 }
+
+const CVector CClientVehicle::GetEntryPoint(int entryPointIndex)
```

Comment:
> Since `entryPointIndex` is non-negative it's better to choose `uint32_t` for it.

`std::uint32_t` to be exact. If we use C++ then lets use C++ headers (and thus `std` namespace)

Reply:
> > Since `entryPointIndex` is non-negative it's better to choose `uint32_t` for it.
> 
> `std::uint32_t` to be exact. If we use C++ then lets use C++ headers (and thus `std` namespace)

std::uint32_t and uint32_t are the same in all contexts. There is no need to use the cumbersome namespace prefix.